### DISCUSSION TITLE
Docker deployment docker deployment rabbitmq: 3.10.7-management cannot set vm_memory_high_watermark.relative through environment variables

### DISCUSSION THREAD

**Discussion by wasai97** [rabbitmq/rabbitmq-server#5829] [Other] [answered]
Docker deployment docker deployment rabbitmq: 3.10.7-management cannot set vm_memory_high_watermark.relative through environment variables

**Comment by michaelklishin** [comment, marked as answer]
FYI, RabbitMQ community [Docker image](https://github.com/docker-library/rabbitmq) is maintained by a different group of people in a separate org.

The image has intentionally dropped support for most environment variables in the last year or two, e.g. see https://github.com/docker-library/rabbitmq/pull/424, https://github.com/docker-library/rabbitmq/pull/501, . Use a custom `rabbitmq.conf`.

**Reply by wasai97** [reply]
Ok, thanks a lot for your reply.

**Comment by wasai97** [comment]
Ok, thanks a lot for your reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "rabbitmq/rabbitmq-server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月20日(星期二) 晚上9:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [rabbitmq/rabbitmq-server] Docker deployment docker deployment rabbitmq: 3.10.7-management cannot set vm_memory_high_watermark.relative through environment variables (Discussion #5829)





 
FYI, RabbitMQ community Docker image is maintained by a different group of people in a separate org.
 
The image has intentionally dropped support for most environment variables in the last year or two, e.g. in docker-library/rabbitmq#501. Use a custom rabbitmq.conf.
